doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
 
 .. _install_mne_c:
 
-Install MNE-C
--------------
+Installing MNE-C
+================
 
-The MNE-C commands can be downloaded
-`here <http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/MNE_register/index.php>`_.
-The :ref:`c_reference` gives an overview of the MNE-C tools.
-
-.. contents:: Contents
+.. contents::
    :local:
    :depth: 1
 
-System requirements and installation
-####################################
-
-The MNE-C runs on Mac OSX and LINUX, and requires:
-
-- Mac OSX version 10.5 (Leopard) or later.
-- LINUX kernel 2.6.9 or later
-- On both LINUX and Mac OSX 32-bit and 64-bit Intel platforms
-  are supported. PowerPC version on Mac OSX can be provided upon request.
-- At least 2 GB of memory, 4 GB or more recommended.
-- Disk space required for the MNE software: 80 MB
-- Additional open source software on Mac OSX, see :ref:`BABDBCJE`.
-
-The MNE software is distributed as a compressed tar archive
-(Mac OSX and LINUX) or a Mac OSX disk image (dmg).
-
-The file names follow the convention:
-
-::
+System requirements
+^^^^^^^^^^^^^^^^^^^
 
-  MNE-* <*version*>*- <*rev*> -* <*Operating system*>*-* <*Processor*>*.* <*ext*>*
```

Comment:
this line (and the explanatory paragraph that followed it) are, in my opinion, a sufficiently widespread practice that there's no need to spell it out here.  Moreover, it's possible to install and use the software without ever knowing what specific version number you've installed, so there's little risk in omitting this info.